<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoLBPStaticVideo</class>
 <widget class="QDialog" name="DemoLBPStaticVideo">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LBP-静态视频分析演示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="emotion_ai_main_window.qrc">
    <normaloff>:/EmotionAI/Resources/app1.png</normaloff>:/EmotionAI/Resources/app1.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>491</width>
     <height>409</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="3" column="3">
       <widget class="QLabel" name="label_vnum">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_22">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>高度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_vheight">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_24">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>帧数</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLabel" name="label_vwidth">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="7">
       <widget class="QLabel" name="label_vpath">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>视频路径</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_vrate">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>帧率</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="loadVideoBtn">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 170, 0);
font: 75 15pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>导入视频</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="playVideoBtn">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 170, 0);
font: 75 15pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>开始分析</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_16">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitDemoBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 170, 0);
font: 75 15pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>退出演示</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="staticVideoLabel">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(222, 221, 220);
font: 75 18pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>LBP Static Video Demo</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>201</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>161</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/sad.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>sadness</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_sa">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/anger.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>anger</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_an">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/fear.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>fear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_fe">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/disgusted.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>disgust</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_di">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/happy.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>happy</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_ha">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/neutral.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>neutral</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_ne">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="emotion_ai_main_window.qrc">:/emolist/Resources/emolist/surprise.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>surprise</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_su">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>154</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="emotion_ai_main_window.qrc">:/EmotionAI/Resources/statistics.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>表情统计</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="emotion_ai_main_window.qrc"/>
  <include location="emotion_ai_main_window.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnLoadVideoButtonClicked()</slot>
  <slot>OnStartVideoBtn_Click()</slot>
 </slots>
</ui>
